src/SFA.DAS.FindApprenticeshipTraining.Api/Models/GetTrainingCourseListItem.cs

```diff
@@ -49,6 +50,16 @@ public class GetTrainingCourseListItem
                 StandardDates = source.StandardDates
             };
         }
+        private static string OrderJobTitles(string jobTitles)
+        {
+            var titles = jobTitles.Split(',').OrderBy(x => x);
+            var titleString = "";
+            foreach (var title in titles)
+            {
+                titleString += title + ",";
+            }
+            return titleString.TrimEnd(',');
```

Comment:
this can be done with string.Join() and will be more efficient than lots of string concat operations.

Reply:
Going back in to make this change